packages/contracts-bedrock/deploy-config/goerli.json

```diff
@@ -1,36 +1,48 @@
 {
-  "finalSystemOwner": "DUMMY",
-  "controller": "DUMMY",
+  "numDeployConfirmations": 1,
+
+  "finalSystemOwner": "0xBc1233d0C3e6B5d53Ab455cF65A6623F6dCd7e4f",
```

Comment:
The `finalSystemOwner` is a Gnosis Safe, see https://goerli.etherscan.io/address/0xBc1233d0C3e6B5d53Ab455cF65A6623F6dCd7e4f

The owners are:

- [0xc37f6a6c4AB335E20d10F034B90386E2fb70bbF5](https://goerli.etherscan.io/address/0xc37f6a6c4AB335E20d10F034B90386E2fb70bbF5)
- [0xA902A27a7631D502E3Ec17fc5d4c3e0861752c94](https://goerli.etherscan.io/address/0xA902A27a7631D502E3Ec17fc5d4c3e0861752c94)
- [0x9bbFB9919062C29a5eE15aCD93c9D7c3b14d31aa](https://goerli.etherscan.io/address/0x9bbFB9919062C29a5eE15aCD93c9D7c3b14d31aa)
- [0x6964f301EdEBF16C563f97cE969C65ECdB39E918](https://goerli.etherscan.io/address/0x6964f301EdEBF16C563f97cE969C65ECdB39E918)

Reply:
Each owner has been funded with 0.5 ETH

Reply:
It is a 3 of 4

Reply:
I would like us to send a test transaction from this gnosis safe before the upgrade tomorrow to 100% guarantee that every party has access to their keys.